src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -171,6 +171,7 @@
 	public static final Block CHESSBOARD = getNull();
 	public static final Block MINI_FROG_STATUE = getNull();
 	public static final Block GLOWYSTONE_DUST = getNull();
+	public static final Block CASSETTE_PLAYER_CLOSED = getNull();
```

Comment:
"Cassette player default" should instead be written as "default cassette player", but even then, I think that the "default" part might be unneccesary here

Reply:
I nammed it "default because I there is different cassette player colors, and we may want to add them someday?

Reply:
Yeah
I'd probably still call it "cassette player", but I suppose that's just up to personal preferrence

Reply:
Ok, I changed it to that !

Reply:
Alright! However you did miss the registry name of the tile entity type

Reply:
Funny thing is, I do thought of it, thinking "hm, wasn't there a registry name for tile entities", but because I didn't found it right away, I just thought that it was my imagination. I corrected it.